app/src/app/poll/poll.component.ts

```diff
@@ -152,6 +156,9 @@ export class PollComponent implements OnInit, OnDestroy {
         next: (evt: PollNotifAnswerEntry) => this.handleEvent(evt),
       }),
     );
+
+    console.log(this.breakpointtablet=getComputedStyle(document.body).getPropertyValue('--breakpoint-tablet'));
+    console.log(this.breakpointlaptop=getComputedStyle(document.body).getPropertyValue('--breakpoint-laptop'));
```

Comment:
Logging informations on the console is very useful while devellopping, but these logs must be removed in the PR.